### ISSUE TITLE
V5 to v6 migration

### ISSUE THREAD

**Issue by Euclidean-Dev** [juanjoDiaz/serverless-plugin-warmup#286]
With v5, it wasn't necessary to define anything more than: 
 ```
warmup:
    default:
        enabled: true
```
Following upgrade to v6 this error was generated during deploy:
```
 Serverless Error ----------------------------------------
     No file matches include / exclude patterns
```
Attempting to resolve this by adding the below to the warmup > default declaration but serverless deploy hung (no error):
```
warmup:
    default:
        enabled: true
        package:
            individually: true
            patterns:
                - "../**"
```
I am guessing something is wrong with the above.  Ideally I think it would be great if v6 could still interpret the most simple v5 config.

**Comment by Herram** [comment]
Same issue. Went back to 5.3.1 and issue resolved itself.
2.66.1 and 2.61.0

**Comment by juanjoDiaz** [comment]
I'm using v2.69.1 and have no problem.
The patterns option was included in [v2.32.0](https://github.com/serverless/serverless/releases/tag/v2.32.0).

Can you create a repo that I can use to reproduce the issue?

**Comment by juanjoDiaz** [comment]
Closing since there is no response and the issue can't be reproduced.
Feel free to reopen if you feel that it's still needed. 🙂
